### ISSUE TITLE
Permission inclusion in areas / Permission lists

### ISSUE THREAD

**Issue by olee** [ForgeEssentials/ForgeEssentialsMain#1564]
The permission system is currently unable to include groups only in an area.
This should be fixed if possible to make it possible to use groups as simple permission lists.

However we need to consider that these checks need to be done in a safe way that ensures the group inclusion is performed at the target of the permission check and not where the player is located at.
If this is not that simple to implement, we should instead or additionally consider adding a special permission-list feature to use these in areas etc.

**Comment by luacs1998** [comment]
Should we combine this with #1810?

**Comment by luacs1998** [comment]
@olee I had a while to think about how to do this.

Should we have a folder where server admins can deposit files with a filename ending in .txt and according to the following syntax:

Example filename: `permlist.txt`

    # Example permission list file.

    test.permission.1=true

    test.permission.2=true

    test.permprop.A=Hello!.

Then in game, the following command can be issued: `/p user TESTER apply permlist`. This would then read from `permlist.txt`(adding the extension automatically) then apply the permissions contained in the list to the player TESTER (or group, or anything else that can have permissions applied to them).

Basically, write the file once, and just apply the file to whatever you want to apply it on.
